examples/with-supertokens/pages/index.js

```diff
@@ -37,9 +31,30 @@ export async function getServerSideProps(context) {
 
 export default function Home(props) {
   return (
-    <ThirdPartyEmailPasswordAuthNoSSR>
-      <ProtectedPage userId={props.userId} />
-    </ThirdPartyEmailPasswordAuthNoSSR>
+    <div className={styles.container}>
+      <Head>
+        <title>SuperTokens 💫</title>
+        <link rel="icon" href="/favicon.ico" />
+      </Head>
+      <main className={styles.main}>
+        <h1 className={styles.title}>
+          Welcome to <a href="https://nextjs.org">Next.js!</a>
+        </h1>
+        <ThirdPartyEmailPasswordAuth requireAuth={true}>
+          <ProtectedPage userId={props.userId} />
```

Comment:
We should showcase the use of useSessionContext as well inside the protected page